tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/indexes/index_name_test.dart

```diff
@@ -119,4 +119,38 @@ void main() {
       );
     });
   });
+
+  test(
+      'Given an index with a name that is longer than 63 characters, then collect an error that the index name is too long.',
+      () {
+    var models = [
+      ModelSourceBuilder().withYaml(
+        '''
+        class: Example
+        table: example
+        fields:
+          name: String
+        indexes:
+          this_is_a_very_long_index_name_that_is_longer_than_63_characters_and_therefore_invalid:
```

Comment:
Question: There is no reason to keep the 56 limit every where? there can never be autogenerated things with field / index names?

Reply:
No, we don't have any issues like this for index names or fields. We do generate a foreign key id field for relations but since that is implicit we can handle that internally, and if the user choses to define it themselves it becomes explicit and the 63 character limit is applied.